assets/univ.js

```diff
@@ -28,7 +28,7 @@ function process_education_input() {
         var course = document.univ_courses[slug_course_in_progress];
         if (course) {
             course.status = 'In progress';
-            course_credits += course.cost || 0;
+            course_credits += parseInt( course.cost ) || 0;
```

Comment:
Intuitively, I'd change this and the second occurrence to `parseInt( course.cost  || 0)`, but I'm not a JS dev :)

Reply:
clearly, i'm not either, as i'd expect 'int += int' to do integer addition rather than concatenation.